aws/provider_test.go

```diff
@@ -73,6 +74,21 @@ func testAccPreCheck(t *testing.T) {
 	}
 }
 
+// testAccAlternateAccountProvider returns the schema.Provider for the alternate account
+// Must be used within a resource.TestCheckFunc
+func testAccAlternateAccountProvider(providers *[]*schema.Provider) (result *schema.Provider) {
```

Comment:
After thinking about it, this is overly tied to one specific case of using alternate accounts. There’s a better way to handle this.